mcodingbot/plugins/stats.py

```diff
@@ -20,6 +20,23 @@
 plugin = Plugin()
 
 
+@plugin.include
+@crescent.command(
+    name="stats", description="Exact values for mCoding statistics"
+)
+async def stats(ctx: Context) -> None:
+    embed = hikari.Embed(
+        title="mCoding stats",
+        color=CONFIG.theme,
+        description=(
+            f"Server members: `{ctx.server.member_count:,}`\n"
+            f"Subscribers: `{_last_known_stats.subs:,}`\n"
+            f"Views: {_last_known_stats.views:,}"
```

Comment:
Also, you didn't put backticks for the last one
```suggestion
            f"Server members: `{ctx.server.member_count:,}`\n"
            f"Subscribers: `{_last_known_stats.subs:,}`\n"
            f"Views: `{_last_known_stats.views:,}`"
```